src/facade/facade_types.h

```diff
@@ -46,6 +49,50 @@ inline std::string_view ToSV(std::string_view slice) {
   return slice;
 }
 
+inline std::string_view ToSV(const std::string& slice) {
+  return slice;
+}
+
+inline std::string_view ToSV(std::string&& slice) = delete;
+
+constexpr auto kToSV = [](auto&& v) { return ToSV(v); };
```

Comment:
```suggestion
inline std::string_view ToSV(const std::string& slice) {
  return slice;
}

inline std::string_view ToSV(std::string&& slice) = delete;

constexpr auto kToSV = [](auto&& v) { return ToSV(std::forward<decltype(v)>(v)); };
```

Reply:
Th expression `ToSV(v)` treats `v` as an lvalue

Reply:
True, missed that one 🤓